v03_pipeline/lib/misc/validation.py

```diff
@@ -69,7 +69,11 @@ def validate_allele_type(
         collected_alleles = sorted(
             [tuple(x) for x in ht.aggregate(hl.agg.collect_as_set(ht.alleles))],
         )
-        msg = f'Alleles with invalid AlleleType are present in the callset: {collected_alleles}'
+        # Handle case where all invalid alleles are NON_REF, indicating a gvcf:
+        if all('<NON_REF>' in alleles for alleles in collected_alleles):
```

Comment:
Maybe?  We could be more tolerant and just proclaim "any" NON_REF is a gVCF.

Reply:
I think the original motivation behind not doing that was that there could be other bad alleles and it would make sense to notify the user of those as well.

Reply:
@matren395 did you have any thoughts on this?